r1cs/core/src/constraint_system.rs

```diff
@@ -46,14 +46,14 @@ pub trait ConstraintSystem<F: Field>: Sized {
 
```

Comment:
Please revert everything you did in this file. Let's not touch this because it may seriously break everything, plus the panic make sense, act as static assertions, no one can trigger a panic and perform DoS attack in this way